Cargo.toml

```diff
@@ -65,9 +65,12 @@ all-features = true
 
 [dependencies]
 atomic = "0.5.0"
-bytes = "1"
-futures = "0.3.1"
-lazy_static = "1.2"
+bytes = "1.1.0"
+futures = "0.3.17"
+futures-timer = "3.0.2"
+getrandom = "0.2.3"
```

Comment:
Do I understand correctly that the crate needs to be listed here in order to enable the `js` feature? If so, would you mind adding a comment? Same for the other crates?